app/components/provider_interface/current_filters_component.html.erb

```diff
@@ -1,17 +1,15 @@
 <div class="moj-filter__selected">
 
   <div class="moj-filter__selected-heading">
-
     <div class="moj-filter__heading-title">
       <h2 class="govuk-heading-m">Selected filters</h2>
     </div>
 
     <div class="moj-filter__heading-action">
     <p class="govuk-body">
-       <%= link_to "Clear", filtering_page_path(params_for_current_state.except(:filter_selections)), class: "govuk-link govuk-link--no-visited-state" %>
+       <%= link_to "Clear", '?' + params_for_current_state.except(:filter_selections).to_query, class: "govuk-link govuk-link--no-visited-state" %>
```

Comment:
We could use `url_for` here. This works, but not in the unit tests for components because the component doesn't know the current URL. In reality the view just knows where they are and `url_for(foo: 'bar')` automatically translates into `/current/url?foo=bar`.

Reply:
🎉 Very nice. Broken component unit tests caused us to introduce `path:` in the first place.

Reply:
Yep that was the reason to begin with - this is nice @tijmenb!